packit_service/service/api/usage.py

```diff
@@ -617,6 +621,92 @@ def get_result_dictionary(
     return {position_name: position, count_name: top_projects.get(project)}
 
 
+@usage_ns.route("/onboarded-projects")
+class Onboarded2024Q1(Resource):
+    @usage_ns.response(
+        HTTPStatus.OK,
+        "Onboarded projects for which exist a Bodhi update or a Koji build or a Packit merged PR.",
+    )
+    @classmethod
+    def calculate(cls):
+        known_onboarded_projects = GitProjectModel.get_known_onboarded_projects()
+
+        bodhi_updates = BodhiUpdateTargetModel.get_all_projects()
+        koji_builds = KojiBuildTargetModel.get_all_projects()
```

Comment:
since we started storing these not so long ago, the number will not be precise :/

but on the other hand, at least we will rely only on our database and not on external API

Reply:
Yes it could be not precise, but on the other hand, if there is a koji build it should be also a merged PR, so we get the project later. This is useful to not do too much work and looking for too many Packit merged PRs.